trusted_applet/main.go

```diff
@@ -301,7 +301,7 @@ func runWithNetworking(ctx context.Context) error {
 		return ctx.Err()
 	}
 
-	go updateChecker(ctx, updateCheckInterval)
+	triggerUpdate := updateChecker(ctx, updateCheckInterval)
```

Comment:
Did you want to increase the value `updateCheckInterval` also?

Reply:
I thought I'd keep that separate so we could agree on what a good value for that should be first :)